spec/requests/questions_spec.rb

```diff
@@ -1,7 +1,12 @@
 require 'rails_helper'
 
 RSpec.describe "Questions", type: :request do
-  describe "GET /index" do
-    pending "add some examples (or delete) #{__FILE__}"
-  end
+  # let(:user) do
+  #   User.create :alias => 'test' :email => 'test@testing.com', :password => '123456', :password_confirmation => '123456'
+  # end
+  # describe "GET /index" do
+  #   it "this retrieves a list of questions" do
+
+  #   end
+  # end
```

Comment:
There aren't any tests here. You should maybe add a test, but if you aren't going to, you can just remove this file.

Reply:
Hi Zan, we started on requesting testing but decided to put it on hold for now. We would like to just keep it since eventually it will be coded out.